src/guides/about-multiprotocol.html

```diff
@@ -55,12 +55,30 @@ <h2 id="resolve-channel-conflict">Resolve channel conflict</h2>
           <li>Re-pair your devices.</li>
         </ol>
       </li>
-      <li><b>Option 2</b>: Use two separate chips, one for each network.
+      <li><b>Option 2</b>: Use two separate wireless chips, one for each network. This works if you have a both Home Assistant
+        Yellow and SkyConnect.
```

Comment:
```suggestion
      <li><b>Option 2</b>: Use two separate wireless chips, one for each network. E.g. by adding a Home Assistant SkyConnect as a second wireless chip.
```

Reply:
What are the alternatives to SkyConnect?